components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -49,8 +48,8 @@ envs {
     envsDirectory = new File(buildDir, 'envs')
     pipInstallOptions = "--trusted-host pypi.python.org --no-cache-dir"
 
-    // Setup a miniconda environment.
-    conda "Miniconda3", "Miniconda3-latest", "64", ["glean_parser==${GLEAN_PARSER_VERSION}"]
+    // Setup a standard python environment.
+    python "python37_64", "3.7.1", ["glean_parser==${GLEAN_PARSER_VERSION}"]
```

Comment:
Good point in fixing the version for the environment. Why are we switching from Miniconda3 to Python? I think it makes sense, but, as you said, we should double check about any weirdness when installing on Windows. I seem to remember (gah, I should have documented that!) that there were issues (e.g. environment not being installed and devs prompted with dialogs) with the barebone environment on Windows :(

**tl;dr** - is this change strictly required in this PR?

Reply:
This change isn't strictly required -- it was just an attempt to reduce disk space usage, since conda is overkill for this purpose.  But obviously, if this doesn't work non-interactively on Windows, that's a non-starter and I can revert it.

Reply:
Yeah, sorry about this, I should have documented my choice in the first place. There's something quirky going on in the environments plugin when the plain python is picked, on Windows. Something that has to do with permissions and stuff, making the installation process interactive (showing the UI), which is undesired. Let's revert this and face the problem as a separate issue, if/when needed.

Reply:
I've reverted and added a comment about why conda is the better choice on Windows.